### PR TITLE
Question dispositif appel

### PR DESCRIPTION
**Pull request by celiacheff**
[Trello](https://trello.com/c/hynNV3ue/436-question-sur-dispositif-dappel)

### CODE REVIEW

File: templates/erps/commune/fields.html [hunk 1]
```diff
@@ -33,6 +33,21 @@ <h4 class="h6 m-0 p-0 font-weight-normal">{{ field.label }}</h4>
...
+        <div class="mb-2">
+          <h4 class="h6 m-0 p-0 font-weight-normal">{{ field.label }}</h4>
+          <small class="text-muted">{{ field.help_text_ui }}</small>
+        </div>
+        <div class="text-right">
+          {% for dispositif in erp.accessibilite.entree_dispositif_appel_type %}
+            <span class="badge badge-success a4a-badge mb-1 cursor-help" title="{{ dispositif|get_dispositifs_appel_label }}">
+          <i aria-hidden="true" class="icon icon-check"></i>
+          {{ dispositif|get_dispositifs_appel_label }}
+        </span><br>
```
Comment (n1k0) [inline templates/erps/commune/fields.html:47]: Watch out indentation here

File: erp/schema.py [hunk 2]
```diff
@@ -67,6 +67,17 @@
...
 
+DISPOSITIFS_APPEL_BOUTON = "bouton"
+DISPOSITIFS_APPEL_SONNETTE = "sonnette"
+DISPOSITIFS_APPEL_INTERPHONE = "interphone"
+DISPOSITIFS_APPEL_VISIOPHONE = "visiophone"
+ENTREE_DISPOSITIFS_APPEL_CHOICES = [
+    (DISPOSITIFS_APPEL_BOUTON, "Bouton d’appel"),
+    (DISPOSITIFS_APPEL_SONNETTE, "Sonnette"),
+    (DISPOSITIFS_APPEL_INTERPHONE, "Interphone"),
+    (DISPOSITIFS_APPEL_VISIOPHONE, "Visiophone")
```
Comment (n1k0) [inline erp/schema.py:78]: Is the `ENTREE_` prefix necessary? If so, why it's not used for every *constants*? I'd vote for ditching the prefix.

File: erp/models.py [hunk 3]
```diff
@@ -956,6 +956,15 @@ class Meta:
         choices=schema.NULLABLE_BOOLEAN_CHOICES,
         verbose_name="Dispositif d'appel",
     )
+    entree_dispositif_appel_type = ArrayField(
+        models.CharField(
+            max_length=255, blank=True, choices=schema.ENTREE_DISPOSITIFS_APPEL_CHOICES
+        ),
+        verbose_name="Équipement(s) sourd/malentendant",
```
Comment (n1k0) [inline erp/models.py:963]: We should use the right label here

File: erp/schema.py [hunk 4]
```diff
@@ -558,6 +569,19 @@
         "nullable_bool": True,
         "warn_if": False,
     },
+    "entree_dispositif_appel_type": {
+        "is_a11y": True,
+        "label": "Type de dispositif d'appel à l'entrée",
+        "help_text": mark_safe(
+            "Quel(s) type(s) de dispositifs sont présents&nbsp;?"
```
Comment (n1k0) [inline erp/schema.py:576]: Don't hesitate to re-precise *Quels types de dispositifs **d'appel** sont présents*

File: erp/schema.py [hunk 5]
```diff
@@ -549,7 +560,7 @@
         "is_a11y": True,
         "label": "Dispositif d'appel à l'entrée",
         "help_text": mark_safe(
-            "Existe-t-il un dispositif comme une sonnette pour permettre à quelqu'un ayant besoin de la rampe ou d'une aide humaine de signaler sa présence&nbsp;?"
+            "Existe-t-il un dispositif pour permettre à quelqu'un signaler sa présence&nbsp;?"
```
Comment (n1k0) [inline erp/schema.py:563]: I'd go for *signaler sa présence **à l'entrée***

**Review by n1k0 — COMMENTED** [review]
Looks good. A few fixes and we're good to go (I'll take care of testing the patch locally though)

File: erp/schema.py (hunk 2 above)
Comment (celiacheff) [reply erp/schema.py:78]: Gotcha, i wanted to keep for consistency, but shortened the internals for sake of brevity.

File: erp/schema.py (hunk 5 above)
Comment (celiacheff) [reply erp/schema.py:563]: As you want :)

**Review by n1k0 — APPROVED** [review]
LGTM

**Pull request merged**